packages/expo-splash-screen/ios/EXSplashScreen/EXSplashScreenConfiguration.h

```diff
@@ -0,0 +1,23 @@
+#import <UIKit/UIKit.h>
+
+typedef NS_ENUM(NSInteger, SplashScreenImageResizeMode) {
```

Comment:
We should probably prefix this by `EX` so it gets versioned properly 🤔 

```suggestion
typedef NS_ENUM(NSInteger, EXSplashScreenImageResizeMode) {
```